[2024-09-17T18:51:11.030+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-17T18:51:11.207+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.extract_data manual__2024-09-17T18:51:09.658061+00:00 [queued]>
[2024-09-17T18:51:11.214+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.extract_data manual__2024-09-17T18:51:09.658061+00:00 [queued]>
[2024-09-17T18:51:11.214+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-17T18:51:11.226+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data> on 2024-09-17 18:51:09.658061+00:00
[2024-09-17T18:51:11.235+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=88) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-17T18:51:11.236+0000] {standard_task_runner.py:72} INFO - Started process 90 to run task
[2024-09-17T18:51:11.237+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_pipeline', 'extract_data', 'manual__2024-09-17T18:51:09.658061+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/NYC_Taxi_pipeline.py', '--cfg-path', '/tmp/tmpj66kr_6n']
[2024-09-17T18:51:11.239+0000] {standard_task_runner.py:105} INFO - Job 202: Subtask extract_data
[2024-09-17T18:51:11.275+0000] {task_command.py:467} INFO - Running <TaskInstance: NYC_Taxi_pipeline.extract_data manual__2024-09-17T18:51:09.658061+00:00 [running]> on host 103d737d0864
[2024-09-17T18:51:11.348+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='NYC_Taxi_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T18:51:09.658061+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-17T18:51:09.658061+00:00'
[2024-09-17T18:51:11.350+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-17T18:51:11.362+0000] {NYC_Taxi_pipeline.py:39} INFO - Data extraction started
[2024-09-17T18:51:11.362+0000] {data_process.py:11} INFO - Looking for Parquet files in folder: ./datalake
[2024-09-17T18:51:11.363+0000] {data_process.py:13} INFO - Found Parquet files: []
[2024-09-17T18:51:11.363+0000] {data_process.py:41} ERROR - Error while extracting trip data: No objects to concatenate
[2024-09-17T18:51:11.363+0000] {NYC_Taxi_pipeline.py:41} INFO - Data extraction done
[2024-09-17T18:51:11.363+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-17T18:51:11.373+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-17T18:51:11.373+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_pipeline, task_id=extract_data, run_id=manual__2024-09-17T18:51:09.658061+00:00, execution_date=20240917T185109, start_date=20240917T185111, end_date=20240917T185111
[2024-09-17T18:51:11.410+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-17T18:51:11.431+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-17T18:51:11.432+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
